### PR TITLE
CoCC: offboard Tim and onboard Carlos

### PR DESCRIPTION
**Pull request by cpanato**
- offboard Tim because now is part of the Steering Committee
-onboard Carlos to CoCC

related to https://github.com/kubernetes/steering/issues/221

/assign @tpepper @cblecker @parispittman

### CODE REVIEW

**Comment by celestehorgan** [comment]
🤔 

/lgtm

File: committee-code-of-conduct/README.md [hunk 1]
```diff
@@ -15,9 +15,9 @@ The [charter](charter.md) defines the scope and governance of the Code of Conduc
 ## Members
 
 * Celeste Horgan (**[@celestehorgan](https://github.com/celestehorgan)**), CNCF
+* Carlos Tadeu Panato Jr. (**[@cpanato](https://github.com/cpanato)**), Mattermost
 * Karen Chu (**[@karenhchu](https://github.com/karenhchu)**), Microsoft
 * Nabarun Pal (**[@palnabarun](https://github.com/palnabarun)**), VMware
-* Tim Pepper (**[@tpepper](https://github.com/tpepper)**), VMware
 * Vallery Lancey (**[@vllry](https://github.com/vllry)**), Apple
```
Comment (liggitt) [inline committee-code-of-conduct/README.md:22]: also update below in the `Term ends on August 7, 2022` section and add Tim to the `Emeritus Committee Members` section

Comment (cpanato) [reply committee-code-of-conduct/README.md:22]: done, thanks for the review!

**Pull request merged**